web/src/main/webapp/script.js

```diff
@@ -12,6 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function searchMe() {
+  var input, filter, ul, li, item, i, txtValue;
+  input = document.getElementById("myInput");
+  filter = input.value.toUpperCase();
+  ul = document.getElementById("task-list");
+  li = ul.getElementsByTagName("li"); 
+   for (i = 0; i < li.length; i++) {
+    item = li[i];
+    txtValue = item.textContent || item.innerText;
+    if (txtValue.toUpperCase().indexOf(filter) > -1) {
+      li[i].style.display = "";
+    } else {
+      li[i].style.display = "none";
+    }
+  }
+}
+
+function loadTasks() {
+  fetch('/list-stock').then(response => response.json()).then((stocks) => {
+    const taskListElement = document.getElementById('task-list');
```

Comment:
Good naming for the variable name, but unclear why this is called a `taskListElement` and not a `stockListElement`